bbq/opcode/print.go

```diff
@@ -133,3 +181,409 @@ func ColorizeOpcode(opcode Opcode) string {
 
 	return aurora.Blue(opcode).String()
 }
+
+// Flow visualization types and constants
+
+// JumpType categorizes different kinds of control flow
+type JumpType int
+
+const (
+	JumpUnconditional JumpType = iota
+	JumpConditional
+	JumpCall
+	JumpReturn
+)
```

Comment:
Maybe prefix these constants with `JumpType` instead of just `Jump`: The package is named `opcode` and defines opcode constants such as `Jump`, `JumpIfFalse`, etc. 
Seeing `JumpUnconditional`, `JumpCall`, etc. might be confusing